workflow-bot-app/src/test/java/com/symphony/bdk/workflow/EventTypesIntegrationTest.java

```diff
@@ -131,6 +133,51 @@ void onMessageReceived_timeout(String swadlFile, String messageToReceive)
         .notExecuted("sendMessageIfNotTimeout");
   }
 
+  @Test
+  void onActivityExpired_timeout() throws IOException, ProcessingException, InterruptedException {
+    final Workflow workflow =
+        SwadlParser.fromYaml(getClass().getResourceAsStream("/event/timeout/activity-expired-with-timeout.swadl.yaml"));
+
+    when(messageService.send(anyString(), any(Message.class))).thenReturn(new V4Message());
+
+    engine.deploy(workflow);
+    engine.onEvent(messageReceived("/start"));
+
+    Thread.sleep(500); // wait 0.5s to let workflow times out
```

Comment:
if we start repeating it let's make it a method (and its name will replace the comment)